src/redux/skybrowser/skybrowserSlice.ts

```diff
@@ -0,0 +1,119 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export interface SkyBrowserImage {
+  cartesianDirection: number[];
+  collection: string;
+  credits: string;
+  creditsUrl: string;
+  dec: number;
+  fov: number;
+  hasCelestialCoords: boolean;
+  identifier: string;
+  key: string;
+  name: string;
+  ra: number;
+  thumbnail: string;
+  url: string;
+}
+
+export interface SkyBrowserBrowser {
+  borderRadius: number;
+  cartesianDirection: number[];
+  color: number[];
+  dec: number;
+  displayCopies: object;
+  fov: number;
+  id: string;
+  isFacingCamera: boolean;
+  isUsingRae: boolean;
+  name: string;
+  opacities: number[];
+  ra: number;
+  ratio: number;
+  roll: number;
+  scale: number;
+  selectedImages: number[];
+  targetId: string;
+}
+
+interface SkyBrowserUpdate
+  extends Pick<SkyBrowserState, 'selectedBrowserId' | 'cameraInSolarSystem'> {
+  browsers: { [id: string]: SkyBrowserBrowser };
+}
+
+export interface SkyBrowserState {
+  isInitialized: boolean;
+  cameraInSolarSystem: boolean;
+  selectedBrowserId: string;
+  selectedBrowser: SkyBrowserBrowser | null;
+  browserIds: string[];
+  browserNames: string[];
+  browserColors: number[][];
```

Comment:
Smae issue here as before. A color as a list of arbitrary number of numbers is too unspecific. 

Do we need a named type for colors?